### PR TITLE
Jormungandr: serialize poi_type in poi from geocodejson

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

**Comment by antoine-de** [comment]
there are some mock bragi responses in the  tests, you should add this field to them

File: source/jormungandr/jormungandr/interfaces/v1/Places.py [hunk 1]
```diff
@@ -199,6 +199,7 @@ def output(self, key, obj):
 
         lon, lat = get_lon_lat(obj)
         geocoding = obj.get('properties', {}).get('geocoding', {})
+        poi_type = geocoding.get('poi_types', [{}])[0]
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/Places.py]: (hmmm, i'm not sure this is possible)
if `geocoding = {..., 'poi_types': [], ...}`,  
then `geocoding.get('poi_types', [{}])[0]` will crash...

Comment (xlqian) [reply source/jormungandr/jormungandr/interfaces/v1/Places.py]: how about `geocoding.get('poi_types', [None])[0]`?

Comment (TeXitoi) [reply source/jormungandr/jormungandr/interfaces/v1/Places.py]: That's the same, if poi_types is [], you'll have [], and boom.

Comment (TeXitoi) [reply source/jormungandr/jormungandr/interfaces/v1/Places.py]: I have a correction for that.

**Comment by TeXitoi** [comment]
@antoine-de test added

**Review by antoine-de — APPROVED** [review]

**Pull request merged**